client/src/store/story.ts

```diff
@@ -1,5 +1,7 @@
-import { string } from '@recoiljs/refine';
-import { atom } from 'recoil';
+'use client';
```

Comment:
The filters belongs to the home page, I would recommend putting all the atoms in the same fille related to the current route. It's just a preference